src/Tribe/Event_Cleaner_Scheduler.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
 /**
- * Class Event_Scheduler
+ * Class Event_Cleaner_Scheduler
  *
  * Uses cron to move old events to trash and/or permanently delete them.
  *
  * @since TBD
  */
-class Tribe__Events__Event_Scheduler {
+class Tribe__Events__Event_Cleaner_Scheduler {
```

Comment:
Why the change in names?

Reply:
I asked for this to make it less ambiguous (since the class exists specifically to deal with cleanup tasks whereas event scheduler is less explicit and could imply it handles publishing events on a schedule or something of that order).